app/main.py

```diff
@@ -1,2 +1,80 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
```

Comment:
The `Node` class is missing the `@dataclass` decorator. Ensure that the `@dataclass` decorator is applied to the `Node` class to automatically generate special methods like `__init__()` and `__repr__()`.